### PR TITLE
CRS-109 SentenceCalculation object

### PR DESCRIPTION
**Pull request by joelstobart-moj**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/SentenceCalculationTest.kt [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    val sentencedAt: LocalDate = LocalDate.of(2020, 1, 1)
+    val sentence = Sentence(duration, sentencedAt, 0, 0)
+    val sentenceCalculation = SentenceCalculation(sentence)
+    val calculatedTotalRemandDays = 0
+    sentenceCalculation.calculatedTotalRemandDays = calculatedTotalRemandDays
+    assertEquals(calculatedTotalRemandDays, sentenceCalculation.calculatedTotalRemandDays)
+  }
+
+  @Test
+  fun licenseExpiryDate() {
```
Comment (yasinmustafa) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/SentenceCalculationTest.kt:66]: minor one but license is spelled licence

File: src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/SentenceCalculationTest.kt [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+import java.time.temporal.ChronoUnit
+
+internal class SentenceCalculationTest {
+
+  @Test
+  fun calculatedReleaseDate() {
```
Comment (yasinmustafa) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/SentenceCalculationTest.kt:11]: minor one, but in kotlin you can name test functions like this in cases where we need to be more descriptive
fun `Calculate a release date`() {

**Review by yasinmustafa — APPROVED** [review]
LGTM - a couple of minor comments

File: src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/SentenceCalculationTest.kt (hunk 1 above)
Comment (joelstobart-moj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/SentenceCalculationTest.kt:66]: Nice spot! :)

**Comment by joelstobart-moj** [comment]
Hi @yasinmustafa  I have addressed those comments

**Review by yasinmustafa — APPROVED** [review]
LGTM

**Pull request merged**